utils/retry/retryexecutor.go

```diff
@@ -0,0 +1,58 @@
+package retry
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"time"
+)
+
+const retryMessage = "%sAttempt %v - %s"
```

Comment:
The convention for constants in go is a leading upper case. Actually, since this template of the message is used only once, maybe we don't need a constant for it.

Reply:
Done